### PR TITLE
Records

### PR DESCRIPTION
**Pull request by msyurikim**

### CODE REVIEW

File: database/postgres/dataGenerator.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    const rateDown = rateOptions[(Math.floor(Math.random() * 3))];
+    const rateUp = (rateDown === null) ? null : !rateDown;
+
+    // writer.write({
+      csvStream.write({
+      // id: announcementCounter++,
+      id: i,
+      title: faker.lorem.sentence().slice(0, 50),
+      postDate: faker.date.recent(),
+      body: faker.lorem.paragraphs().slice(0, 1000),
```
Comment (ingridclaire) [inline database/postgres/dataGenerator.js:43]: this is a good idea. I need to check that my paragraphs have a char limit.

File: database/postgres/dataGenerator.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  const announcementsCount = Math.floor((Math.random() * 10) + 1);
+  var start = new Date().getTime();
+  for (var i = 0; i < 10000000; i++) {
+    writer.write({
+      gameNumber: i,
+      name: faker.lorem.words().slice(0, 30),
+      image: faker.image.image(),
+      announcements: announcements,
+      url: faker.internet.url()
+    });
```
Comment (ingridclaire) [inline database/postgres/dataGenerator.js:72]: Are your headers for the csv table populating automatically?

File: database/postgres/index.js [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+//       (err, data) => {
+//         if (err) {
+//           callback(err);
+//         } else {
+//           callback(null, data);
+//         }
+//       },
+//     );
+//   },
+// };
```
Comment (ingridclaire) [inline database/postgres/index.js:161]: did you bring this to postgres file for reference? or is this the old file renamed?

Comment (msyurikim) [reply database/postgres/index.js:161]: I copied it in from the original database file for reference.

**Pull request closed without merging**